poleno/utils/tests/test_templatetags.py

```diff
@@ -199,18 +199,31 @@ def language_view(request):
             u'request': request,
         })))
 
-    urls = tuple(patterns(u'',
+    def page_not_found(request):
+        return HttpResponseNotFound(Template(
+            u'{% load change_lang from poleno.utils %}'
+            u'({% change_lang "de" %})'
```

Comment:
By sa zislo, aby tu boli vsetky 3 jazyky. Nech je to symetricke s `language_view`. A nech vidime, ze `change_lang` funguje pre vsetky jazyky. Aj pre aktivny aj pre neaktivny.

Reply:
:ok: